### DISCUSSION TITLE
Your uploaded proof has already been used previously.

### DISCUSSION THREAD

**Discussion by YiHui-Liu** [community/community#109804] [GitHub Education] [answered]
### Hi everyone,

I have upload certification of student status information by Ministry of Education of the People's Republic of China, but i get:
- You are unlikely to be verified until you have completed your [GitHub billing information](https://github.com/settings/billing/payment_information) with your full name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields. You do not have to add a payment method. You may need to log out and log back in to GitHub before re-applying.
- Try applying again after making more [contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) to the GitHub Classroom assignment(s) created by your teacher.
- Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

I modified the filename as suggested by others and got:
- Your uploaded proof has already been used previously.

**Comment by Songmg-Nwafu** [comment]
I'm facing the same problem. How do you deal with it?

**Reply by YiHui-Liu** [reply, marked as answer]
Since the relosution of my labtop camera is too low, so i use the front camera of my phone (although it can find all the camera of my phone, but i can only use the front one) to take a photo of the laptop screen, and pass the verification.

**Reply by Songmg-Nwafu** [reply]
Thanks~  According to the way of you, I have already submitted the application.
